iommi/style.py

```diff
@@ -90,7 +92,8 @@ def __init__(self, *bases, base_template=None, content_block=None, assets=None,
                     self.content_block = base.content_block
                     break
 
-        self.assets = {k: v for k, v in Namespace(*(base.assets for base in bases), assets).items() if v is not None}
+        self.assets = {k: (Asset(**v) if isinstance(v, Namespace) else v)
+                       for k, v in Namespace(*(base.assets for base in bases), assets).items() if v is not None}
```

Comment:
There is probably a more elegant way to express this.  Maybe one should have a `AssetContainer` class, that is `Traversable` and self.assets should be an instance of that.  Or something.